cookbooks/subduction_initiation/doc/subduction_initiation.md

```diff
@@ -0,0 +1,91 @@
+(sec:cookbooks:subd-init)=
+# Subduction initiation from Matsumoto and Tomoda (1983)
+
+*This section was contributed by Cedric Thieulot.*
+
+The setup for this experiment originates in {cite:t}`matsumoto:tomoda:1983`.
+
+In this very early computational geodynamics paper the authors
+are interested in predicting the future process of crustal and lithospheric movement
+at the gigantic fracture zone in the Northeastern
+Pacific by means of numerical simulation
+of contact between two kinds of viscous fluid of different density.
+They then solve the incompressible isothermal linear Stokes equations
+with a stream function approach and the resulting equation is solved by means of the
+Finite Difference method and the Marker and Cell (MAC) method.
+
+The domain is a 2D Cartesian box of size $L_x \times L_y=(400~\text{ km},180~\text{ km})$.
+There are three fluids in the domain: water ($\rho_w=1030~\text{ kg m}^{-3}$, $\eta_w=10^{-3}~\text{ Pa s}$),
+lithosphere ($\rho_l=3300~\text{ kg m}^{-3}$, $\eta_l$) and
+asthenosphere ($\rho_a=3200~\text{ kg m}^{-3}$, $\eta_a$).
+Note that although the water viscosity is correct, this is probably not
+the value that was used in the authors since it would most likely lead to
+numerical errors. We then set $\eta_w=10^{19}~\text{ Pa s}$ (we can then speak of 'sticky water').
```

Comment:
```suggestion
numerical errors. We thus set $\eta_w=10^{19}~\text{ Pa s}$ (we can then speak of 'sticky water').
```

Reply:
:-)

Honey?

Reply:
sticky honey is a hard sell ... is there a planet covered in honey with plate tectonics?